### PR TITLE
Improve error reporting to show locators and more context

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ module.exports = function drivex(driver, wd) {
               if (found !== false) {
                 d.fulfill(found.click());
               } else {
-                d.reject(new Error('couldn\'t find option with text: ' + optionText));
+                d.reject(new Error('[driverx.selectByOptionText] couldn\'t find option with text: ' + JSON.stringify(optionText) + ' for locator ' + showLocator(locator)));
```
Comment (grawk) [inline index.js]: small typo on this line

**Review by grawk — COMMENTED** [review]
Looks good aside from small typo in one of the error messages. Approval pending that correction.

File: index.js (hunk 1 above)
Comment (aredridel) [reply index.js]: I'm not seeing the typo?

Comment (grawk) [reply index.js]: `driverx` instead of `drivex`. sorry i should have specified it was in the error string

**Comment by gabrielcsapo** [comment]
👍 apart from @grawk's comment.

**Comment by aredridel** [comment]
Fixed!

**Comment by grawk** [comment]
published as `v1.0.14`

**Comment by aredridel** [comment]
\o/

**Pull request merged**